djangoplicity/templates/admin/base_site_base.html

```diff
@@ -21,7 +21,7 @@
 {% endblock %}
 
 {% block nav-global %}
-<div class="djangoplicity-tools"><a href="/">{% trans 'Website' %}</a>
+<div class="djangoplicity-tools"><a href="{{ DJANGOPLICITY.URL_PREFIX }}/">{% trans 'Website' %}</a>
```

Comment:
Do you have this already configured with a default value in djangoplicity?.

Reply:
yes, in a previous PR, we added this constant, in the context processor on based in Noirlab settings